spark-bigquery-dsv1/spark-bigquery-with-dependencies_2.12/src/test/java/com/google/cloud/spark/bigquery/acceptance/Scala212DataprocImage20AcceptanceTest.java

```diff
@@ -13,22 +13,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.spark.bigquery.acceptance;
+package com.google.cloud.spark.bigquery.acceptance;
 
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
 
-public class DataprocImage14AcceptanceTest extends DataprocAcceptanceTestBase {
+public class Scala212DataprocImage20AcceptanceTest extends DataprocAcceptanceTestBase {
 
   private static AcceptanceTestContext context;
 
-  public DataprocImage14AcceptanceTest() {
+  public Scala212DataprocImage20AcceptanceTest() {
     super(context);
   }
 
   @BeforeClass
   public static void setup() throws Exception {
-    context = DataprocAcceptanceTestBase.setup("2.11", "1.4-debian10");
+    context = DataprocAcceptanceTestBase.setup("2.0-debian10");
```

Comment:
If we are keeping the enum DataprocImage, then we can get this String from the enum